libs/angular/src/lib/context-menu/context-menu.component.html

```diff
@@ -0,0 +1,31 @@
+<button class="ghost small" (click)="open($event)">
+  <i class="sg-icon sg-icon-ellipsis">Open context menu</i>
+</button>
+
+<div
+  class="popover popover-context-menu"
+  [class.active]="isActive"
+  [style.top]="top"
+  [style.left]="left"
+  #contextMenuPopover
+>
+  <ul role="listbox">
+    <li
+      *ngFor="let menuItem of menuItems; let i = index"
+      (click)="onItemClick(menuItem)"
+      role="option"
+      (keydown)="onMenuItemKeyDown($event, i)"
```

Comment:
this could pass menuItem instead index,  by which menu item is retrieved in ts file